filterpage.html

```diff
@@ -0,0 +1,372 @@
+              <label for="Mahindra">Mahindra</label>
+            </li>
+            <li class="checkbox-wrapper">
+              <input type="checkbox" id="Toyota" value="Toyota" />
+              <label for="Toyota">Toyota</label>
+            </li>
+            <li class="checkbox-wrapper">
+              <input type="checkbox" id="Mercedes-Benz" value="Mercedes-Benz" />
+              <label for="Mercedes-Benz">Mercedes-Benz</label>
+            </li>
+          </ul>
+
+          <ul>
+            <li class="checkbox-wrapper">
+              <input type="checkbox" id="Hyundia" value="Hyundia" />
+              <label for="Hyundia">Hyundia</label>
+            </li>
+            <li class="checkbox-wrapper">
+              <input type="checkbox" id="Tata" value="Tata" />
+              <label for="Tata">Tata</label>
+            </li>
+            <li class="checkbox-wrapper">
+              <input type="checkbox" id="Kia" value="Kia" />
+              <label for="Kia">Kia</label>
+            </li>
+            <li class="checkbox-wrapper">
+              <input type="checkbox" id="Honda" value="Honda" />
+              <label for="Honda">Honda</label>
+            </li>
+          </ul>
+        </div>
+      </div>
+
+      <div class="body-wrapper">
+        <div class="js-body-and-arrow-wrapper">
+          <h2 class="body-heading">Body Type</h2>
+          <img
+            src="icons8-more-than-48.png"
+            alt="Upwards arrow"
+            class="js-body-arrow"
+          />
+        </div>
+
+        <div class="js-body-list-wrapper js-collapse">
+          <ul class="body-list-wrapper--content">
+            <li class="checkbox-wrapper">
+              <input type="checkbox" id="SUV" value="SUV" />
+              <label for="SUV">SUV</label>
+            </li>
+            <li class="checkbox-wrapper">
+              <input type="checkbox" id="Hatchback" value="Hatchback" />
+              <label for="Hatchback">Hatchback</label>
+            </li>
+          </ul>
+
+          <ul>
+            <li class="checkbox-wrapper">
+              <input type="checkbox" id="Sedan" value="Sedan" />
+              <label for="Sedan">Sedan</label>
+            </li>
+            <li class="checkbox-wrapper">
+              <input type="checkbox" id="CompactSUV" value="Compact SUV" />
+              <label for="CompactSUV">Compact SUV</label>
+            </li>
+          </ul>
+        </div>
+      </div>
+
+      <button class="js-apply-filters__btn">Apply Filters</button>
+    </div>
+
+    <script>
+      let selectedMakeFilter = [];
+      let selectedBodyFilter = [];
+      let openFilterButton = document.querySelector(".js-open-filter__btn");
+      let selection = document.querySelector(".js-selection-wrapper");
+      let openFirstPageWrapper = document.querySelector(
+        ".js-open-filter-page-wrapper"
+      );
+      openFilterButton.addEventListener("click", function () {
```

Comment:
if any function can return null or undefined value for those function we add null check

Reply:
done